src/cljs/org/broadinstitute/firecloud_ui/common/gcs_file_preview.cljs

```diff
@@ -18,12 +17,13 @@
       {:header "File Details"
        :content
        (let [{:keys [data error status]} (:response @state)
-             data-size (when data (data "size"))
+             data-size (:size data)
```

Comment:
Yes, if `data` is `nil`, then `(:size data)` will be `nil`.  One of the perks of working with keyword-keyed maps.